WcaOnRails/config/routes.rb

```diff
@@ -115,6 +115,8 @@
 
   get '/render_markdown' => 'markdown_renderer#render_markdown'
 
+  get '/update_locale/:locale' => 'application#update_locale', as: :update_locale
```

Comment:
(Gonna get up on my REST horse here) This sounds like it has a side effect, shouldn't it be a POST or PATCH or something?